### PR TITLE
Update to Wine 9.9 development to support wayland native.

### PR DESCRIPTION
**Pull request by sHedC**
Update to Wine 9.9 development to support wayland native.

Move Config check ealier to allow for env settings in config file better.

Fix Icon showing intask bar when started from menu

Allow wine version fix in docker build (fixed to 9.9)

### CODE REVIEW

File: update_zwift.sh [hunk 1]
```diff
@@ -58,11 +58,17 @@ function wait_for_zwift_game_update() {
 
 if [ ! "$(ls -A .)" ] # is directory empty?
 then
+    # Prevent Wine from trying to install a different mono version
+    WINEDLLOVERRIDES="mscoree,mshtml=" wineboot -u
```
Comment (netbrain) [inline update_zwift.sh:62]: :partying_face:

File: zwift.sh [hunk 2]
```diff
@@ -36,6 +36,24 @@ msgbox() {
...
+    source $HOME/.config/zwift/config
+fi
+
+# Check for $USER specific zwift configuration, sourced here and passed on to container aswell
+if [[ -f "$HOME/.config/zwift/$USER-config" ]]
+then
+    ZWIFT_USER_CONFIG_FLAG="--env-file $HOME/.config/zwift/config"
+    source $HOME/.config/zwift/config
+fi
+
```
Comment (netbrain) [inline zwift.sh]: Guess this needs to be updated/rebased from #142

**Review by netbrain — APPROVED** [review]
Lgtm

File: update_zwift.sh (hunk 1 above)
Comment (sHedC) [reply update_zwift.sh:62]: Not sure if we even need mono to be installed at all?  As we install dot.net which overrides it right?

File: zwift.sh (hunk 2 above)
Comment (sHedC) [reply zwift.sh]: Done

File: update_zwift.sh (hunk 1 above)
Comment (netbrain) [reply update_zwift.sh:62]: no wine mono is required atleast it was in all my previous testing.

**Review by netbrain — APPROVED** [review]

**Comment by netbrain** [comment]
Ill merge this and build zwift from scratch which will from this point onward create a new zwift image with wine 9.9.

:crossed_fingers:

**Comment by netbrain** [comment]
https://github.com/netbrain/zwift/actions/runs/9299444600

File: update_zwift.sh (hunk 1 above)
Comment (sHedC) [reply update_zwift.sh:62]: I will try

**Comment by sHedC** [comment]
I'll try without mono I think it was needed before because it auto installs on wineboot.  But will  check :)

**Comment by sHedC** [comment]
Build failed I ahve seen that before it is getitng a previous version from the web site and updating to the current version in this case zwift is 1.65 from the website and 1.66 is the actual.  

I updated the line to not use cache but still :(

**Comment by sHedC** [comment]
I am updateing to add a better check that checks that one version is equal to or greater than

**Pull request merged**